proto/config/netinst.proto

```diff
@@ -97,14 +116,29 @@ message NetworkInstanceConfig {
   //    itself is not created on the device.
   bool activate = 5;
 
-  // port - Only a single port is supported.
-  //    This is used as the external connection for the network instance.
-  //    This can be a physical (eth0 ) or logical port (vlan 0).
-  //    The port name comes from DeviceConfig ( When it is supported in future).
-  //    If the user needs multiple physical ports, Device config should be
-  //    used to create a label for multiple physical ports.
+  // Port(s) providing external connectivity for the network instance.
+  // Either a single port referenced by its name (SystemAdapter.Name) or an adapter label
+  // matching multiple network ports (ports containing this label in SystemAdapter.Labels).
```

Comment:
The label can match zero or more ports - matching one port is quite common e.g., for "uplink" when there is only one management port.

Reply:
Comment updated.